src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -206,5 +227,28 @@ private string GetApplicationInsightsKey()
 
             return Environment.GetEnvironmentVariable("ApplicationInsights__InstrumentationKey");
         }
+
+        private void IncludeXmlComments(SwaggerGenOptions options)
+        {
+            string fileName = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            string fullFilePath = Path.Combine(AppContext.BaseDirectory, fileName);
+            options.IncludeXmlComments(fullFilePath);
+            string fullFilePathApi = Path.Combine(AppContext.BaseDirectory, "Altinn.App.Api.xml");
+            options.IncludeXmlComments(fullFilePathApi);
+        }
+
+        private string GetApplicationId()
+        {
+            try
+            {
+                string appMetaDataString = File.ReadAllText("config/applicationmetadata.json");
+                JObject appMetadataJObject = JObject.Parse(appMetaDataString);
+                return appMetadataJObject.SelectToken("id").Value<string>();
+            }
+            catch (Exception e)
```

Comment:
You can remove the parameter if it's not being used.
```suggestion
            catch
```

Reply:
Added logging of the exception.